Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM denoland/deno
```

Comment:
I suggest specifying a tag, like `denoland/deno:1.44.2`, so that builds are more reproducible.

Reply:
Agreed. Resolved in `eab1fd4`.